.eslintrc.json

```diff
@@ -18,5 +17,9 @@
     "camelcase": "off",
     "no-param-reassign": "warn",
     "arrow-parens": ["error", "as-needed"]
-  }
+  },
+  "extends": [
+    "airbnb-base",
+    "prettier"
```

Comment:
Why?

Reply:
There is conflict between eslint and prettier. e.g. add line break before/after operators

Reply:
Can't reproduce. Is there a concrete example?

Reply:
Nevermind. I think it's good to have it either way.

Reply:
For the highlighted line, it exceed maximum line length and has to break into 2 lines. Eslint require the plus sign to be at the front of line 10 whereas Prettier requires it at the back of line 9.
![operator_linebreak](https://user-images.githubusercontent.com/45759668/68073635-155c3c00-fdcd-11e9-8d5f-476a5dd207ee.png)